packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -189,7 +189,13 @@ export const noIndexSetting = new PublicInstanceSetting<boolean>('noindex', fals
 export const verifyEmailsSetting = new PublicInstanceSetting<boolean>("verifyEmails", true, "optional");
 
 export const hasCuratedPostsSetting = new PublicInstanceSetting<boolean>("hasCuratedPosts", false, "optional");
+
 export const hasSideCommentsSetting = new PublicInstanceSetting<boolean>("comments.sideCommentsEnabled", isLWorAF, "optional");
 export const hasCommentsTableOfContentSetting = new PublicInstanceSetting<boolean>("comments.tableOfContentsEnabled", isLWorAF, "optional");
 export const hasDialoguesSetting = new PublicInstanceSetting<boolean>("dialogues.enabled", true, "optional");
 export const hasPostInlineReactionsSetting = new PublicInstanceSetting<boolean>("posts.inlineReactionsEnabled", isLWorAF, "optional");
+
+/** Whether datadog is enabled for this forum (not necessarily for the current user */
+export function isDatadogEnabledOnSite() {
+  return forumTypeSetting.get() === 'EAForum';
```

Comment:
Could you make this `isEAForum`? I think we're trying to remove these string checks